docs/guide/ops/catalog/index.md

```diff
@@ -302,7 +302,14 @@ The `brooklyn` CLI includes several commands for working with the catalog.
 * `--catalogInitial <file.bom>` will set the catalog items to use on first run,
   on a catalog reset, or if persistence is off
 
-If [persistence](../persistence/) is enabled, catalog additions will remain between runs.
+If `--catalogInitial` is not specified, the default initial catalog at `brooklyn/default.catalog.bom` will be used.
```

Comment:
Worth mentioning (or focusing on) that the default catalog will scan the classpath for catalog items.